app/views/pages/instances/create.liquid

```diff
@@ -0,0 +1,17 @@
+---
+method: post
+layout: instances
+slug: instances
+---
+
+{% liquid
+  include 'require_auth'
+
+  function object = 'lib/commands/instances/create', domain: context.params.domain, pos_cli_token: context.params.pos_cli_token, available: true 
```

Comment:
A better way would probably be to use `instance` namespace for params, so you can provide `object: context.params.instance` instead of each parameter individually